app.js

```diff
@@ -7,17 +7,18 @@ const path = require('path');
 const engine = require('ejs-locals');
 
 const app = express();
-const config = require('./config');
-const routes = require('./app/routes');
+app.use(express.static(path.join(__dirname, 'public')));
+
+const config = require('./config/config.example');
```

Comment:
1) Why will u require an example file? Never do that.
2) It's fine to reference the folder app/routes when u have an index.js file, you don't have to explicitly mention /index.